libs/clients/cbr/client.go

```diff
@@ -86,6 +123,23 @@ type CredentialsIssueResponse struct {
 	SignedTokens []string `json:"signed_tokens"`
 }
 
+// GetIssuerV2 - get a v2 issuer from CB
```

Comment:
isn't this used to get v3 issuers? :thinking:

Reply:
we were attempting to re-use the existing issuer get endpoint, which admittedly causes much confusion.  Will correct.

Reply:
https://github.com/brave-intl/challenge-bypass-server/pull/292

Reply:
https://github.com/brave-intl/bat-go/pull/1705